packages/pangea-node-sdk/src/utils/utils.ts

```diff
@@ -1,4 +1,5 @@
 import CryptoJS from "crypto-js";
+import * as crypto from "crypto";
```

Comment:
This doesn't need to be done now, but seems we should be able to replace the use of `CryptoJS` with Node's `crypto` module.

Reply:
Yes, I don't like the idea of using both. I'll try to fix it in the future